src/index.hbs

```diff
@@ -0,0 +1,11 @@
+{{> _includes/header}}
```

Comment:
My preference would be that we move the main HTML tags into this template and have the partials contain the markup within those sections.  

For example:

```
<html>
  <head>
    {{> _includes/header}}
  </head>
  <body>
    <!-- ... ->
    {{> _includes/footer}}
  </body>
</html>
```